### PR TITLE
Add CFML and FreeSpec.

### PR DESCRIPTION
**Pull request by Zimmi48**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +84,7 @@ The [Coq proof assistant](https://coq.inria.fr) provides a formal language to wr
 
 ### Tools
 
+- [CFML](https://gitlab.inria.fr/charguer/cfml) - Tool for proving OCaml programs in Separation Logic.
```
Comment (palmskog) [inline README.md]:
```suggestion
- [CFML](https://gitlab.inria.fr/charguer/cfml) - Tool for proving properties of OCaml programs in separation logic.
```
For consistency with other mentions of SL.

File: README.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ The [Coq proof assistant](https://coq.inria.fr) provides a formal language to wr
 ### Frameworks
 
 - [Fiat](https://github.com/mit-plv/fiat) - Mostly automated synthesis of correct-by-construction programs.
+- [FreeSpec](https://github.com/ANSSI-FR/FreeSpec) - Framework to modularly verify programs with effects and effect handlers.
```
Comment (palmskog) [inline README.md]:
```suggestion
- [FreeSpec](https://github.com/ANSSI-FR/FreeSpec) - Framework for modularly verifying programs with effects and effect handlers.
```

**Comment by palmskog** [comment]
In the interest of time for this low-hanging fruit, I'll just merge this and do a separate commit with my proposed changes.

**Comment by Zimmi48** [comment]
Sorry, I'm not very reactive on GitHub these days.

**Pull request merged**